### PR TITLE
Webpack configuration

### PR DESCRIPTION
**Pull request by RaoAkif**
Create an src folder and add index.html, index.js & style.css files in to it.
- Initialize the project using
npm init -y
- Add webpack.config.js.
- install Webpack packages using
npm install webpack webpack-cli --save-dev
- Run CSS Loader Command
npm install --save-dev style-loader css-loader
-Install  HTML-Webpack-Plugin
npm i html-webpack-plugin
- Add gitIgnore file and ignore node_modules and dist folders

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "license": "ISC",
+  "devDependencies": {
+    "css-loader": "^6.7.1",
+    "style-loader": "^3.3.1",
+    "webpack": "^5.70.0",
+    "webpack-cli": "^4.9.2"
+  },
+  "dependencies": {
+    "html-webpack-plugin": "^5.5.0"
+  }
```
Comment (mikethreels) [inline package.json]: - [ ] according to the requirements  Webpack dev server should be set up and running with no errors.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Rao, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
- [ ] please make sure to use[ the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (RaoAkif) [reply package.json]: @mikethreels Would you please be specific about the `no errors`. I couldn't get it.

Comment (RaoAkif) [reply package.json]: Any ScreenShot of Errors that you are talking about?

Comment (mikethreels) [reply package.json]: Hi Rao,
I just put the entire requirement in this comment, what I'm aiming at is the fact that you don't have "webpack-dev-server" setup at all, it should be a part of your project dependencies.

Comment (RaoAkif) [reply package.json]: Ohhh. OK. That was the issue, I was unable to understand. Thanks for the suggestion.

**Review by Lordkaito — APPROVED** [review]
Hi @RaoAkif, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**